{"name":"Get: Failure","fullName":"TestManagerPgRepoSuiteRunner/ManagerPgRepoSuite/TestRequestPgRepo_GetFailure","status":"passed","statusDetails":{"message":"","trace":""},"start":1728207674844,"stop":1728207674844,"uuid":"21139c16-83c7-11ef-8b51-be2bab8c44a2","historyId":"fdc6f0b12c80b5e756bc51abb0f5a3ce","testCaseId":"746276940ad85cbf47dc7d8cd6ea856d","labels":[{"name":"language","value":"go1.23.0"},{"name":"framework","value":"Allure-Go@v0.6.0"},{"name":"host","value":"MBP-Pavel-2.netis.cc"},{"name":"thread","value":"TestManagerPgRepoSuiteRunner/ManagerPgRepoSuite/TestRequestPgRepo_GetFailure"},{"name":"suite","value":"ManagerPgRepoSuite"},{"name":"package","value":"cookdroogers/internal/repo/pg"},{"name":"tag","value":"ManagerPgRepo"}],"steps":[{"name":"Failure","status":"passed","start":1728207674844,"stop":1728207674844,"steps":[{"name":"ASSERT: Nil","status":"passed","start":1728207674844,"stop":1728207674844,"parameters":[{"name":"Actual","value":"*models.Manager((*models.Manager)(nil))"}]},{"name":"ASSERT: Error Is","status":"passed","start":1728207674844,"stop":1728207674844,"parameters":[{"name":"Error","value":"sql: connection is already closed: error in postgres repo"},{"name":"Target","value":"error in postgres repo"}]}]}]}